src/pages/Friend/FriendPage.jsx

```diff
@@ -0,0 +1,258 @@
+import styled from 'styled-components';
+import { useState, useEffect, useRef } from 'react';
+import { arrayChop } from '../../components/common/CarouselComponent';
+import CarouselComponent from '../../components/common/CarouselComponent';
+import BottomSheetComponent from '../../components/common/BottomSheetComponent';
+import Button from '../../components/common/Button';
+import VerticalCard from '../../components/Friend/VerticalCard';
+import HorizontalCard from '../../components/Friend/HorizontalCard';
+import { ReactComponent as IcnUnion } from '../../assets/Friend/icn_union.svg';
+
+// const data = null;
+const data = [
+  {
+    nickname: '닉네임1',
+    birthday: '3월 18일',
+  },
+  {
+    nickname: '닉네임2',
+    birthday: '5월 14일',
+  },
+  {
+    nickname: '닉네임999999999999999999999999',
+    birthday: '12월 21일',
+  },
+];
+
+const FriendPage = () => {
+  const chopedDataList = data && arrayChop(data, 2);
+  const [bottomSheetShow, setBottomSheetShow] = useState(false);
+  const inputRef = useRef(null);
+  const [email, setEmail] = useState('');
+
+  // 바텀시트 렌더링 시 input 포커스
+  useEffect(() => {
+    inputRef.current?.focus();
+  });
+
+  return (
+    <SLayout>
+      <SH1 as='header'>친구</SH1>
+      {data && (
+        <SSection>
+          <ST1>나에게 선물한 친구</ST1>
+          <CarouselComponent pageLength={chopedDataList.length} pageWidth={335}>
+            {chopedDataList.map((item, index) => (
+              <SPageContainer key={index}>
+                <VerticalCard data={item[0]} />
+                {index === chopedDataList.length - 1 &&
+                data.length % 2 !== 0 ? (
+                  <div style={{ visibility: 'hidden' }}>
+                    <VerticalCard />
+                  </div>
+                ) : (
+                  <VerticalCard data={item[1]} />
+                )}
+              </SPageContainer>
+            ))}
```

Comment:
```
  const CarouselContent = chopedDataList.map((item, index) => (
    <SPageContainer key={index}>
      <VerticalCard data={item[0]} />
      {index === chopedDataList.length - 1 && data.length % 2 !== 0 ? (
        <div style={{ visibility: 'hidden' }}>
          <VerticalCard />
        </div>
      ) : (
        <VerticalCard data={item[1]} />
      )}
    </SPageContainer>
  ));
```
```suggestion
            {CarouselContent}
```

구조 좀 더 잘 보이게 수정하자면 캐러셀이랑 바텀시트 내용물을 따로 변수로 작성하는 것도 좋을 것 같아요!